lib/analyzer/binaries-analyzer.ts

```diff
@@ -7,15 +7,23 @@ export {
   analyze,
 };
 
-async function analyze(targetImage: string) {
-  const binaries = await getBinaries(targetImage);
+async function analyze(targetImage: string, installedPackages: string[]) {
+  let binaries: Binary[] = [];
+  if (!installedByPackageManager(installedPackages)) {
+    binaries = await getBinaries(targetImage);
+  }
   return {
     Image: targetImage,
     AnalyzeType: 'binaries',
     Analysis: binaries,
   };
 }
 
+function installedByPackageManager(installedPackages: string[]): boolean {
+  return installedPackages
+    .filter((pkg) => pkg === 'node' || pkg === 'nodejs').length > 0;
```

Comment:
Is there a way we can compare the results of the binary and system results so we don't need to keep manually updating this function?

Something like, get the binaries, then filter anything that is in the OS deps list

Reply:
Let me work it out...

Reply:
:crown: